content/en/docs/setup/install/istioctl/index.md

```diff
@@ -456,6 +459,10 @@ Some parameters will temporarily exist in both the Helm and `IstioOperator` APIs
 namespaces and enablement settings. The Istio community recommends using the `IstioOperator` API as it is more
 consistent, is validated, and follows the [community graduation process](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE-CHECKLIST.md#feature-lifecycle-checklist).
 
+{{< tip >}}
+Helm API may not support some kubernetes settings, please use [Customize Kubernetes settings](#customize-kubernetes-settings)
+{{< /tip >}}
```

Comment:
i suggest omitting this, since we already mention it at the start of the doc.

Reply:
done

Reply:
Doesn't look done. Remove lines 462-465.

Reply:
ok forgot to commit, now it is done